### PR TITLE
Updated free-programming-interactive-tutorials-en.md

### PR DESCRIPTION
**Pull request by Bisma-Nadeemm**
learn python step by step easily

## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo

## For resources
Programming Tutorials ,Python Tutorials, DSA, OS  

### Description
Programiz is an online educational platform that provides a wide range of programming tutorials, resources, and tools to help individuals learn, practice, and master programming and related topics.

### Why is this valuable (or not)?
It is free of cos and very easy to use

### How do we know it's really free?
It doesn't demand any money

## Checklist:
- [x ] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by eshellman** [comment]
Be sure to check the free-programming-books-lint output for linter errors. Alphabetical order and spacing are the most common problems.

File: more/free-programming-interactive-tutorials-en.md [hunk 1]
```diff
@@ -371,6 +371,7 @@
 * [How to Think Like a Computer Scientist: Learning with Python, Interactive Edition](http://interactivepython.org/courselib/static/thinkcspy/index.html)
 * [Learn Python](http://www.learnpython.org)
 * [Learn Python Step by Step](http://www.techbeamers.com/python-tutorial-step-by-step)
+* [Learn Python](https://www.programiz.com/python-programming) - Programiz
```
[the remark this answers is not in the document]
Comment (Bisma-Nadeemm) [reply more/free-programming-interactive-tutorials-en.md]: Ok

**Comment by Ajay-singh1** [comment]
LGTM!Ready to Merge

**Comment by eshellman** [comment]
duplicate -  listed in the courses list

**Pull request closed without merging**